pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
Not sure why you see that error when building the docs, as the `klass` kwarg should be included in the `_shared_doc_kwargs` for NDFrame as well.

But still, since this is not redefined in series/dataframe, you always get `NDFrame` in the docstring (for both Series/DataFrame.describe), so this is a reason to get it out of the `_shared_docs` IMO